lib/logstash/filters/geoip.rb

```diff
@@ -119,6 +119,9 @@ class LogStash::Filters::GeoIP < LogStash::Filters::Base
   # to having multiple caches for different instances at different points in the pipeline, that would just increase the
   # number of cache misses and waste memory.
   config :lru_cache_size, :validate => :number, :default => 1000
+  
+  # Tags the event on failure to look up geo information. This can be used in later analysis.
+  config :tag_on_failure, :validate => :array, :default => ["_geoip_lookup_failure"]
```

Comment:
Wanted this to be consistent with grok filter https://github.com/logstash-plugins/logstash-filter-grok/blob/master/lib/logstash/filters/grok.rb#L204

I thought about the array value vs string. I think I understand why this is an array -- you could have additional tags based on hostname where this is running or something like that for further debugging. 

IMO configurable is nice, users can tweak it